osu.Game.Rulesets.Osu/Mods/OsuModBarrelRoll.cs

```diff
@@ -30,6 +31,8 @@ public class OsuModBarrelRoll : Mod, IUpdatableByPlayfield, IApplicableToDrawabl
         public override string Description => "The whole playfield is on a wheel!";
         public override double ScoreMultiplier => 1;
 
+        public override string SettingDescription => $"{SpinSpeed.Value}rpm {Direction.Value.GetDescription().ToLowerInvariant()}";
```

Comment:
rpm is a unit of measure, so it needs a space.

```suggestion
        public override string SettingDescription => $"{SpinSpeed.Value} rpm {Direction.Value.GetDescription().ToLowerInvariant()}";
```